kusto/query_builder.go

```diff
@@ -507,35 +507,68 @@ func NewStmt(query stringConstant, options ...StmtOption) Stmt {
 	return s
 }
 
+// AddDatabase will add a normalized Database string to the Stmt. This is similar to the + operator on two strings,
```

Comment:
// AddDatabase will add a normalized Database string to the Statement..
// Given a variable with a value 'MyDatabase', AddDatabase(arg) will produce the following string '["MyDatabase"]'

For all below

Reply:
not accurate as this describe NormalizeName and not really these functions. i can maybe write 
// AddDatabase  - Given a variable with a value 'MyDatabase', AddDatabase(arg) will produce the following string '["MyDatabase"]' and will add it to a given stmt

Reply:
"Given ... will append the following string to Stmt